### PR TITLE
rsgroup sagroup create

### PR DESCRIPTION
**Pull request by sandy4877**
please review my pr

### CODE REVIEW

File: main/main.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    name = var.rg_name
+    location = var.location
+    tags = var.tags
+}
+
+resource "azurerm_storage_account" "rg" {
+  name                     = "sandystorage4877"
+  resource_group_name      = azurerm_resource_group.rg.name
+  location                 = azurerm_resource_group.rg.location
+  account_tier             = "Standard"
```
Comment (techslateramu) [inline main/main.tf:14]: why is this hard coded, move this to variables

File: main/main.tf [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  account_replication_type = "GRS"
```
Comment (techslateramu) [inline main/main.tf:15]: same here - move to variables

**Review by techslateramu — COMMENTED** [review]
fix the review comments and commit the code to same branch again

File: main/main.tf (hunk 1 above)
Comment (sandy4877) [reply main/main.tf:14]: fixed

File: main/main.tf (hunk 2 above)
Comment (sandy4877) [reply main/main.tf:15]: fixed

File: main/main.tf (hunk 1 above)
Comment (techslateramu) [reply main/main.tf:14]: you have not fixed... you need to move this into varialbes.tf... this applies to all !

**Review by techslateramu — COMMENTED** [review]
you have not fixed them.. please fix them... 

you need to move them into variables and manage well !

**Pull request merged**